desktop/apps/collect_art/routes.coffee

```diff
@@ -4,7 +4,7 @@ resizer = require '../../components/resizer'
 markdown = require '../../components/util/markdown'
 Items = require '../../collections/items'
 
-landing = new JSONPage name: 'buy-art'
+landing = new JSONPage name: 'collect-art'
```

Comment:
Has the name of the file in S3 been changed as well? Otherwise this may be a breaking change.